index.ts

```diff
@@ -1067,58 +1071,51 @@ commander
-        if (functionName) {
-          functionName = functionName.padEnd(15);
-        } else {
-          functionName = ERROR.NO_FUNCTION_NAME;
-        }
-        let payload = '';
+        functionName = functionName ? functionName.padEnd(15) : ERROR.NO_FUNCTION_NAME;
+        let payloadData = '';
         if (cmd.json) {
-          payload = JSON.stringify(entries[i], null, 4);
+          payloadData = JSON.stringify(entries[i], null, 2);
         } else {
-          if (payloadType === 'textPayload') {
-            payload = entries[i].metadata.textPayload;
-          } else if (payloadType === 'jsonPayload') {
-            payload = entries[i].metadata.jsonPayload.fields.message.stringValue;
-          } else if (payloadType === 'protoPayload') {
-            payload = entries[i].metadata.protoPayload;
-          } else {
-            payload = ERROR.PAYLOAD_UNKNOWN;
-          }
-          if (payload && typeof(payload) === "string") {
-            payload = payload.padEnd(20);
+          payloadData = ({
+            textPayload: metadata.textPayload,
+            jsonPayload: metadata.jsonPayload ? metadata.jsonPayload.fields.message.stringValue : '',
+            protoPayload: metadata.protoPayload
+          })[payload] || ERROR.PAYLOAD_UNKNOWN;
+
+          if (payloadData && typeof(payloadData) === 'string') {
+            payloadData = payloadData.padEnd(20);
           }
         }
-        const logString = `${severity.padEnd(10)} - ${functionName} - ${payload} - [${timestamp}]`;
-        console.log(logString);       
+        let coloredSeverity = ({
+          ERROR: chalk.red(severity),
+          INFO: chalk.blue(severity),
+          DEBUG: chalk.yellow(severity)
+        })[severity] || severity;
+        coloredSeverity = String(coloredSeverity).padEnd(15);
+        console.log(`${coloredSeverity} ${timestamp} ${functionName} ${payloadData}`);   
       }
     }
   
-    getProjectSettings().then(({ scriptId, rootDir, projectId }: ProjectSettings) => {
+    getProjectSettings().then(({ projectId }: ProjectSettings) => {
       if (!projectId) {
         console.error(ERROR.NO_GCLOUD_PROJECT);
-        process.exit(-1);
-      } else {
-        if (cmd.open) {
-          const stackdriverURL = `https://console.cloud.google.com/logs/viewer?project=${projectId}&resource=app_script_function`;
-          open(stackdriverURL);
-          process.exit(0);
-        }
-        const logging = new Logging({
-            projectId: projectId
-        });
-        logging.getEntries().then((entryData: any) => {
-          printLogs(entryData[0]);
-        });
+        return process.exit(-1);
       }
-    });
+      if (cmd.open) {
+        const stackdriverURL = `https://console.cloud.google.com/logs/viewer?project=${projectId}&resource=app_script_function`;
+        open(stackdriverURL);
+        process.exit(0);
+      }
+      const logger = new logging({ projectId });
+      logger.getEntries().then(printLogs);
+      });
```

Comment:
Nit: Indentation

Reply:
Fixed.

Reply:
This doesn't fix the indentation.
It should be like this:
```ts
  const logger = new logging({ projectId });
  logger.getEntries().then(printLogs);
});
```

(I'll have to make sure Travis catches this)

Reply:
Good catch, should be actually fixed now.